example/download_vcc2016data.py

```diff
@@ -62,6 +63,10 @@ def download(url, dest=None):
 
 
 if __name__ == "__main__":
+    print(
+        "Warning: This script was replaced with download_corpus_data.py and"
```

Comment:
I reccomend to modify the sentence as following:
"Warning: This script was replaced by download_corpus_data.py and will be removed soon."